src/main/java/org/jahia/modules/rss/RSSUtil.java

```diff
@@ -72,6 +73,30 @@
 
     private String url;
 
+    private static int timeoutInMs = 4000; // default value -> to be loaded from Jahia properties? (global timeout config)
+    
+    /**
+     * Get a SyndFeed from an url
+     * @param url
+     * @return
+     */
+    public static SyndFeed loadSyndFeed(String url){
+        SyndFeed feed = null;
+        try {
+            final SyndFeedInput input = new SyndFeedInput();
+            for loading RSS feeds. * User: ktlili
+ * Date: Jan 5, 2010
```

Comment:
There seems to be a problem with the commit, this will not compile correctly. Could you please check and fix this ?